public/assets/css/navbar_admin.css

```diff
@@ -0,0 +1,66 @@
+:root{
+    --full-relative: 100%;
+    --zero: 0;
+}
+
+.navAdmin .navbar {
+    position: relative;
+}
+
+.navAdmin::before {
+    --border-height: 6px;
+    position: absolute;
+    top: var(--zero);
+    content: ' ';
+    width: var(--full-relative);
+    height: var(--border-height);
+    background-color: var(--primary-color);
+}
+
+.navAdmin .navbar-nav {
+    width: var(--full-relative);
+    justify-content: space-around;
+}
+
+.navAdmin .btn-nav-admin {
+    --width-btn: 90px;
+    --height-btn: 35px;
+    --radius-btn: 100px;
```

Comment:
Honnest question : So, that means these buttons are the only ones in your whole project that are going to have these properties, right?

Reply:
yes ok review this with my team